src/crypto/store/indexeddb-crypto-store.js

```diff
@@ -314,7 +314,7 @@ export class IndexedDBCryptoStore {
      * @param {string} type A key type
      */
     async getCrossSigningPrivateKey(txn, func, type) {
-        this._backend.getCrossSigningPrivateKey(txn, func, type);
+        return this._backend.getCrossSigningPrivateKey(txn, func, type);
```

Comment:
Oh, I think I misunderstood how these work...? Looks like the actual data is passed to `func`, so perhaps the `return` wasn't needed before or now...?

Reply:
Also I guess this could drop `async` now...?

Reply:
mm, I suppose it could